Gruntfile.js

```diff
@@ -3,35 +3,62 @@
  * https://github.com/wildbit/grunt-postmark.git
  */
 
-module.exports = function(grunt) {
+module.exports = function (grunt) {
+  var secret = grunt.file.readJSON('secrets.json');
+  var config = grunt.file.readJSON('config.json');
 
   grunt.initConfig({
-    secrets: grunt.file.readJSON('secrets.json'),
+    secret: secret,
+    config: config,
 
     /* Postmark
-    ------------------------------------------------- */
+     ------------------------------------------------- */
     postmark: {
       options: {
-        serverToken: '<%= secrets.serverToken %>'
+        serverToken: "<%= secret.postmark.server_token %>",
       },
       email: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
-        subject: 'Yo',
+        from: "<%= config.postmark.from %>",
+        to: "<%= config.postmark.to %>",
+        subject: "<%= config.postmark.subject %>",
         src: ['test/email.html']
       },
       bulk: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
-        subject: 'Hey',
+        from: "<%= config.postmark.from %>",
+        to: "<%= config.postmark.to %>",
+        subject: "<%= config.postmark.subject %>",
         src: ['test/*.html']
       }
-    }
+    },
+
+    // you can either specify the template configuration here, or in templates.json
+    'postmark-templates-upload': config.templates && config.templates
```

Comment:
Optionally storing the template data in a separate JSON file is a good idea, but this might be a roadblock for people who are configuring this for the first time.

`example_config.json` references `templates.json` so I ran into issues during setup because it wasn't clear whether the plugin was using the data from the template config or from the target config.

Instead of reading from `templates.json` we should remove this and document it as an optional step. This would reduce the friction during the initial setup process.